### PR TITLE
Update default ciphers used by kube-rbac-proxy

### PR DESCRIPTION
**Pull request by paulfantom**
Cross-referenced ciphers from Intermediate list in https://wiki.mozilla.org/Security/Server_Side_TLS with currently supported ones in https://github.com/golang/net/blob/master/http2/ciphers.go and https://golang.org/src/crypto/tls/cipher_suites.go

### CODE REVIEW

File: jsonnet/kube-prometheus/kube-prometheus.libsonnet [hunk 1]
```diff
@@ -108,32 +108,12 @@ local configMapList = k3.core.v1.configMapList;
...
-      'TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256',
-      'TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256',
-
-      // disabled by h2 means: https://github.com/golang/net/blob/e514e69ffb8bc3c76a71ae40de0118d794855992/http2/ciphers.go
-
-      // 'TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384',   // TODO: Might not work with h2
-      // 'TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384', // TODO: Might not work with h2
-      // 'TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305',    // TODO: Might not work with h2
-      // 'TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305',  // TODO: Might not work with h2
+      'TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256',
```
Comment (brancz) [inline jsonnet/kube-prometheus/kube-prometheus.libsonnet:111]: do you mind if we keep each of these and continue to comment why the rest can't/shouldn't be used?

Comment (paulfantom) [reply jsonnet/kube-prometheus/kube-prometheus.libsonnet]: Done :)

**Comment by paulfantom** [comment]
Tested on OpenShift 4.6

**Comment by brancz** [comment]
Thank you!

**Pull request merged**